### DISCUSSION TITLE
Collect all CSharp relevant examples

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [k2-fsa/sherpa-onnx#717] [General]
# C# API
https://k2-fsa.github.io/sherpa/onnx/csharp-api/index.html

https://github.com/k2-fsa/sherpa-onnx/tree/master/dotnet-examples

**Comment by GeorgeS2019** [comment]
Question:

which of the examples show Audio to phoneme using onnx

https://github.com/k2-fsa/sherpa-onnx/issues/716#issuecomment-2026951900

**Reply by csukuangfj** [reply]
We only have audio to text (i.e., audio to words).

There is no audio to phoneme in sherpa-onnx.
